.travis.yml

```diff
@@ -2,14 +2,20 @@ language: java
 sudo: false
 jdk:
   - oraclejdk8
-
-install: mvn clean package 
+install: mvn --settings settings.xml clean verify
 script: 
   - mkdir tmp/ 
   - export PROJECT_VERSION=`mvn -q -Dexec.executable="echo" -Dexec.args='${project.version}' --non-recursive exec:exec` 
   - tar -C tmp -xvf modules/dist.cli/target/dist.cli-${PROJECT_VERSION}.tar.gz 
   - tmp/bin/cmclient --version | grep -q "^${PROJECT_VERSION}" 
-
 cache:
   directories:
   - $HOME/.m2
+before_install:
+- openssl aes-256-cbc -K $encrypted_d5d7c7a681ca_key -iv $encrypted_d5d7c7a681ca_iv
```

Comment:
The private key used for code signing is contained in an encrypted format in the project sources. Before we can use this private key the key needs
* to be decrypted
* put into the keychain